### PR TITLE
AG-7967 - Axis domain docs page

### PR DESCRIPTION
**Pull request by manapeirov**
PR for https://ag-grid.atlassian.net/browse/AG-7967

### CODE REVIEW

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes-domain/examples/axis-nice/main.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      position: "left",
+      title: {
+        text: "Market Share (%)",
+      },
+      nice: true
+    },
+  ],
+  legend: {
+    enabled: false,
+  },
```
Comment (alantreadway) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes-domain/examples/axis-nice/main.ts:39]: Is this redundant in light of your other change to disable legend with <=1 series?

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes-domain/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+The axis domain is the extent of displayed values along the axis.
+
+For a continuous axis, such as the [Number](/charts-axes-types/#number) or [Time](/charts-axes-types/#time) axis, the domain is calculated automatically from the minimum and maximum values of the data.
+
+For the [Category](/charts-axes-types/#category) axis, the domain consists of the discrete values in the data.
+
+## Nice Domain
+
+By default, for a continuous axis, the domain is extended to have nice start and stop values.
```
Comment (alantreadway) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes-domain/index.md:13]: Maybe (briefly!) elaborate on what `nice` means exactly, as it's not clear from the prose here.

**Review by alantreadway — COMMENTED** [review]
LGTM other than

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes-domain/index.md (hunk 2 above)
Comment (manapeirov) [reply grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes-domain/index.md]: done

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**